README.md

```diff
@@ -171,6 +171,214 @@ rs = [dataset.to_table(nearest={"column": "vector", "k": 10, "q": q})
 | [java](./java)     | Java bindings (JNI)      |
 | [docs](./docs)     | Documentation source     |
 
+## Java JNI Bindings
```

Comment:
Hi, I feel this section is too long as our project readme. Maybe it's better to keep them in the jni's own readme?